Mathlib/Condensed/Light/Epi.lean

```diff
@@ -119,14 +119,13 @@ instance : (lim (J := Discrete ℕ) (C := LightCondMod R)).PreservesEpimorphisms
   preserves f _ := by
     have : lim.map f = (Pi.isoLimit _).inv ≫ Limits.Pi.map (f.app ⟨·⟩) ≫ (Pi.isoLimit _).hom := by
       apply limit.hom_ext
-      intro ⟨n⟩
+      intro
       simp only [lim_obj, lim_map, limMap, IsLimit.map, limit.isLimit_lift, limit.lift_π,
         Cones.postcompose_obj_pt, limit.cone_x, Cones.postcompose_obj_π, NatTrans.comp_app,
         Functor.const_obj_obj, limit.cone_π, Pi.isoLimit, Limits.Pi.map, Category.assoc,
         limit.conePointUniqueUpToIso_hom_comp, Pi.cone_pt, Pi.cone_π, Discrete.natTrans_app,
         Discrete.functor_obj_eq_as]
-      erw [IsLimit.conePointUniqueUpToIso_inv_comp_assoc]
-      rfl
+      exact (IsLimit.conePointUniqueUpToIso_inv_comp_assoc _ _ _ _).symm
```

Comment:
I haven't checked myself, but I'm assuming the goal doesn't really look much like the statement of `IsLimit.conePointUniqueUpToIso_inv_comp_assoc`?
So now this `exact` is hiding defeq abuse...

The reason we want to get rid of `erw` is that it is usually abusing defeqs. And the defeq abuse is the real problem. So when removing an `erw`, I would like to make sure that the defeq abuse is also resolved.